pkg/client/example-runner.go

```diff
@@ -55,10 +68,17 @@ func RunExamplesForService(registry broker.BrokerRegistry, client *Client, servi
 
 }
 
+type CompleteServiceExample struct {
+	Example                    broker.ServiceExample   `json: "service_example"`
+	ServiceName                string                  `json: "service_name"`
+	ServiceId                  string                  `json: "service_id"`
+	ServiceBindOutputVariables []broker.BrokerVariable `json: "broker_variables"`
```

Comment:
WDYT about renaming this to something like `ExpectedOutput` to better indicate what its purpose is?